pkg/backup/abs/abs.go

```diff
@@ -89,15 +90,23 @@ func (w *ABS) Put(key string, r io.Reader) error {
 	blobName := path.Join(v1, w.prefix, key)
 	blob := w.container.GetBlobReference(blobName)
 
+	var buf bytes.Buffer
+	if r != nil {
+		_, err := io.Copy(&buf, r)
```

Comment:
This will also return io.EOF error which is not an error, you should handle the error using ReadFrom instead

Reply:
Check out here. https://github.com/Azure/azure-sdk-for-go/blob/master/storage/blockblob.go#L116-L133

This is how go sdk handle this stuff. I want to use their practice.

Reply:
Ok this is what readfrom does, I am ok if you want to copy what sdk does :)